external/include/libcxx/thread

```diff
@@ -400,6 +420,15 @@ template <class _Fp>
 void* __thread_proxy_cxx03(void* __vp)
 {
     std::unique_ptr<_Fp> __p(static_cast<_Fp*>(__vp));
+#if defined(__TINYARA__)
```

Comment:
Is it reqd to keep redundant code ? can we keep this logic in another func and call from both apis?

Reply:
Ok. I will add a new function for common code.